richtext-ui/src/commonMain/kotlin/com/halilibo/richtext/ui/string/RichTextString.kt

```diff
@@ -50,6 +50,26 @@ public class RichTextStringStyle(
   public val codeStyle: SpanStyle? = null,
   public val linkStyle: TextLinkStyles? = null
 ) {
+  public fun copy(
+    boldStyle: SpanStyle? = null,
```

Comment:
why take null as the default parameter then resolve to member value?

In this case if I have a `RichTextStringStyle` with a non-null `boldStyle`, I cannot copy and make it `null`. It will always resolve to its current value.

```
public fun copy(
  boldStyle: SpanStyle? = this.boldStyle,
  ...
)
```

would be more appropriate.

Reply:
Good catch. Just fixed.